package.json

```diff
@@ -25,5 +25,8 @@
   "version": "0.1.0",
   "devDependencies": {
     "webpack-dev-server": "^3.11.2"
+  },
+  "scripts": {
+    "build": "bundle exec rake assets:precompile"
```

Comment:
I don't really see a compelling reason to have `yarn build` over just calling `bundle exec rake assets:precompile` directly. A developer using this already needs to run `bundle install` and this just feels like another level of indirection.

I don't feel super strongly about this, but if there is no compelling reason then I'd say lets remove this.

Reply:
It's definitely unnecessary. I removed it.